### PR TITLE
#1163 - add option to turn off hamburger menu

### LINKED ISSUE #1163: [connect > portal] UI - Update hamburger menu
**Task**
Create a different flavor of the hamburger menu in which the menu items are reorganized and no longer within the hamburger menu. This means that the render of the menu items (Bridge, Resume Transaction, FAQ, and Terms of Use) are going to be represented in a different part of the widget, which will improve the discoverability of these features.

**Acceptance Criteria**
- Bridge (hide this, as this is going to be irrelevant with the "removal" of the hamburger menu)
- Move Resume Transaction to the main page of the widget
- Move FAQ and Terms of Use to the bottom of the widget
- Add link for "advanced dev tools" that directs user to former Portal Bridge
- Hide branding text at bottom (Portal Bridge only) "Want to embed the Wormhole Connect bridge into your own dapp or website?
Do it in as few as three lines of code here: https://wormhole.com/connect"

**Design**
- Wireframe:

Image

**Note**
- Hide FAQ on Portal Bridge main site since connect has its own FAQ in hamburger --- associated ticket (https://github.com/orgs/XLabs/projects/2/views/6?pane=issue&itemId=42688988)

### PR DESCRIPTION
**Pull request by sebastianscatularo**
image

### CODE REVIEW

File: wormhole-connect/src/components/FooterNavBar.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import { ICON } from 'utils/style';
+import { Route, setRoute } from 'store/router';
+
+const useStyles = makeStyles()(() => ({
+  menuIcon: ICON,
+  menu: {
+    display: 'flex',
+    flexDirection: 'row',
+    gap: '8px',
+    padding: '8px',
```
Comment (M-Picco) [inline wormhole-connect/src/components/FooterNavBar.tsx:13]: May I suggest a few tweaks for this css to space the items around a bit more uniformly?

```suggestion
    padding: '8px',
    width: '85%',
```

File: wormhole-connect/src/components/FooterNavBar.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  },
+  menuItem: {
+    padding: '16px 0',
+    textAlign: 'left',
```
Comment (M-Picco) [inline wormhole-connect/src/components/FooterNavBar.tsx]:
```suggestion
    textAlign: 'center',
    flex: 1,
```

image

Dismiss this comment and the previous one if this is the intended design

**Review by M-Picco — COMMENTED** [review]
I think the config is not being applied to the Redeem page:

https://github.com/wormhole-foundation/wormhole-connect/assets/11276821/d4167fbb-96d9-488d-a1bc-4c0ba4c2c16d

File: wormhole-connect/src/components/FooterNavBar.tsx (hunk 2 above)
Comment (sebastianscatularo) [reply wormhole-connect/src/components/FooterNavBar.tsx]: we do not have a design defined yet for this, so, I'll update this with your suggestions

File: wormhole-connect/src/components/FooterNavBar.tsx (hunk 1 above)
Comment (sebastianscatularo) [reply wormhole-connect/src/components/FooterNavBar.tsx:13]: applied

File: wormhole-connect/src/components/FooterNavBar.tsx (hunk 2 above)
Comment (sebastianscatularo) [reply wormhole-connect/src/components/FooterNavBar.tsx]: applied

**Comment by sebastianscatularo** [comment]
this was addressed https://github.com/wormhole-foundation/wormhole-connect/pull/1180#pullrequestreview-1723375907

**Review by M-Picco — DISMISSED** [review]

**Review by M-Picco — APPROVED** [review]

**Pull request merged**